docker/test/fasttest/run.sh

```diff
@@ -185,6 +185,7 @@ function clone_submodules
             contrib/dragonbox
             contrib/fast_float
             contrib/NuRaft
+            contrib/llvm
```

Comment:
Fast test will become slow after this change.
If we can build without LLVM, it's better to keep fast test lightweight.

Reply:
Fixed.